src/examples/cancel/Shootist.java

```diff
@@ -56,18 +76,12 @@
 
     private boolean requestTerminated;
 
-    private static Logger logger = Logger.getLogger(Shootist.class);
-
-
-
-
-
     public void processRequest(RequestEvent requestReceivedEvent) {
         Request request = requestReceivedEvent.getRequest();
         ServerTransaction serverTransactionId = requestReceivedEvent
                 .getServerTransaction();
 
-        logger.info("\n\nRequest " + request.getMethod()
+        LOG.info("\n\nRequest " + request.getMethod()
```

Comment:
I will try my best to reduce it as much as possible, as I removed in plenty of files the manual logger config I am quite unsure if the amount of files with changes will go down a lot, however let's hope for the best. I will ping you again once I've made those changes.

Concerning code-style: I would propose to maven-ize the project first and then introduce a common code style (which is enforced by the maven-checkstyle-plugin).

BR Stefan